packages/components/addon/components/hds/tabs/tab-panel.hbs

```diff
@@ -0,0 +1,11 @@
+{{! Hds-TabPanel }}
+<section
+  class="hds-tab-panel"
+  role="tabpanel"
+  {{!-- id="section1" --}}
+  {{!-- aria-labelledby="tab1" --}}
+  {{! hidden TODO: set by arg }}
```

Comment:
I commented out all attributes with non-static values. Next step is to add these back in with underlying JavaScript.